packages/studio/src/components/AddPageButton/EntityIdField.tsx

```diff
@@ -85,7 +83,7 @@ export default function EntityIdField({
       </div>
       <div className="flex items-center mt-2">
         <select
-          className="disabled:bg-gray-100 border border-r-0 border-gray-400 text-gray-900 focus:ring-blue-500 focus:border-blue-500 py-2 pl-1 pr-1"
+          className="disabled:bg-gray-100 border border-r-0 border-gray-400 text-gray-900 focus:ring-blue-500 focus:border-blue-500 py-2 pl-1 pr-1 text-sm h-[39px] rounded-l-lg"
```

Comment:
nit: I think the height should be 38px, not 39px?

Reply:
For some reason 39px seemed to fit better, I think due to the border width?

Reply:
At 100% zoom I prefer 39px but when I zoom in 38px lines up better
I'll just use 38 for now